prime-router/docs/api/waters-delivery.yml

```diff
@@ -85,7 +85,7 @@ paths:
           description: unauthorized
         '404':
           description: report does not exist
-  /waters/deliveries/{deliveryId}/facilities:
+  /waters/report/{id}/facilities:
```

Comment:
Is this a breaking change to the API?  Is anything else using it?

Reply:
That endpoint is being added in this PR, so it should be fine. Just updating the doc now that the endpoint is gonna exist.